src/shared/infrastructure/profile-metrics/profile-metrics.json

```diff
@@ -43,28 +43,28 @@
   },
   "memory:alloc_objects:count:space:bytes": {
     "id": "memory:alloc_objects:count:space:bytes",
-    "description": "Total objects allocated",
+    "description": "Number of objects allocated",
     "type": "alloc_objects",
     "group": "memory",
     "unit": "short"
   },
   "memory:alloc_space:bytes:space:bytes": {
     "id": "memory:alloc_space:bytes:space:bytes",
-    "description": "Total bytes allocated in the heap",
+    "description": "Size of memory allocated in the heap",
```

Comment:
Do we need to state that the number is in bytes?

Reply:
It seems to be the confusing part (mentioned in the linked GitHub issue):
> We should not use the SI prefix, as the axis scales it to the fitting SI prefix and that's confusing

Reply:
Okay. We don't have to specify bytes. What does "SI prefix" refer to?

Reply:
Oh I think I misunderstood: https://www.nist.gov/pml/owm/metric-si-prefixes

We might want to specify the unit - but ensure we don't mention the prefix ("kilo", "mega", ..)